build/templates/definition.xml.erb

```diff
@@ -11,8 +11,8 @@
   <PublisherName>Chef Software, Inc.</PublisherName>
   <LocalResources />
   <IsInternalExtension><%= is_internal %></IsInternalExtension>
-  <Eula>http://www.getchef.com/about/</Eula>
-  <PrivacyUri>http://www.getchef.com/about/</PrivacyUri>
-  <HomepageUri>http://www.getchef.com/</HomepageUri>
+  <Eula>http://www.chef.io/about/</Eula>
+  <PrivacyUri>http://www.chef.io/about/</PrivacyUri>
+  <HomepageUri>http://www.chef.io/</HomepageUri>
   <IsJsonExtension>true</IsJsonExtension>
```

Comment:
@andrewelizondo 
Azure made some changes in publishing API, we have to change order of xml element here. 
`<PublisherName>Chef Software, Inc.</PublisherName>` moved below to `<IsJsonExtension>true</IsJsonExtension>` .